src/lib/api/tools.ts

```
@@ -0,0 +1,136 @@
+
+function getSortOrder(sort: SortOption): Prisma.ToolOrderByWithRelationInput {
+    switch (sort) {
+        case 'newest':
+            return { createdAt: 'desc' }
+        case 'oldest':
+            return { createdAt: 'asc' }
+        case 'updated':
+            return { lastCommit: 'desc' }
+        case 'forks':
+            return { forks: 'desc' }
+        case 'stars':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)